# Fix for:
# ```
# <inline asm>:1:41: error: expected '%<type>' or "<type>"
# .pushsection ".debug_gdb_scripts", "MS",@progbits,1
# ```
# See:
# json: https://github.com/boostorg/json/pull/1064
# unordered: https://github.com/boostorg/unordered/pull/295
# interprocess: https://github.com/boostorg/interprocess/pull/246
# outcome: https://github.com/boostorg/outcome/commit/eae4849ebceb4c967dfc14cf39c3ab1a8cb30d34#diff-a8472b48cf892c2567d924bc452849a6214c48a001a383093908bafcea61ce80R725
#
--- boost_1_87_0/boost/unordered/unordered_printers.hpp	2024-12-05 06:23:37.000000000 +0530
+++ boost_1_87_0.mod/boost/unordered/unordered_printers.hpp	2025-01-28 18:54:08.263552974 +0530
@@ -13,7 +13,7 @@
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Woverlength-strings"
 #endif
-__asm__(".pushsection \".debug_gdb_scripts\", \"MS\",@progbits,1\n"
+__asm__(".pushsection \".debug_gdb_scripts\", \"MS\",%progbits,1\n"
         ".ascii \"\\4gdb.inlined-script.BOOST_UNORDERED_UNORDERED_PRINTERS_HPP\\n\"\n"
         ".ascii \"import gdb.printing\\n\"\n"
         ".ascii \"import gdb.xmethod\\n\"\n"
--- boost_1_87_0/boost/interprocess/interprocess_printers.hpp	2024-12-05 06:23:32.000000000 +0530
+++ boost_1_87_0.mod/boost/interprocess/interprocess_printers.hpp	2025-01-28 18:53:42.712756801 +0530
@@ -13,7 +13,7 @@
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Woverlength-strings"
 #endif
-__asm__(".pushsection \".debug_gdb_scripts\", \"MS\",@progbits,1\n"
+__asm__(".pushsection \".debug_gdb_scripts\", \"MS\",%progbits,1\n"
         ".ascii \"\\4gdb.inlined-script.BOOST_INTERPROCESS_INTERPROCESS_PRINTERS_HPP\\n\"\n"
         ".ascii \"import gdb.printing\\n\"\n"
 
--- boost_1_87_0/boost/json/detail/gdb_printers.hpp	2024-12-05 06:23:33.000000000 +0530
+++ boost_1_87_0.mod/boost/json/detail/gdb_printers.hpp	2025-01-28 18:53:25.848839810 +0530
@@ -23,7 +23,7 @@
 #endif
 
 __asm__(
-  ".pushsection \".debug_gdb_scripts\", \"MS\",@progbits,1\n"
+  ".pushsection \".debug_gdb_scripts\", \"MS\",%progbits,1\n"
   ".ascii \"\\4gdb.inlined-script.BOOST_JSON_DETAIL_GDB_PRINTERS_HPP\\n\"\n"
   ".ascii \"import gdb\\n\"\n"
   ".ascii \"import gdb.printing\\n\"\n"
--- boost_1_87_0/boost/outcome/experimental/status-code/status_code.hpp	2024-12-05 06:23:35.000000000 +0530
+++ boost_1_87_0.mod/boost/outcome/experimental/status-code/status_code.hpp	2025-01-28 18:53:07.504814720 +0530
@@ -722,7 +722,7 @@
 #pragma clang diagnostic ignored "-Woverlength-strings"
 #endif
 __asm__(
-".pushsection \".debug_gdb_scripts\", \"MS\",@progbits,1\n"
+".pushsection \".debug_gdb_scripts\", \"MS\",%progbits,1\n"
 ".ascii \"\\4gdb.inlined-script.BOOST_OUTCOME_SYSTEM_ERROR2_INLINE_GDB_PRETTY_PRINTERS_H\\n\"\n"
 ".ascii \"import gdb.printing\\n\"\n"
 ".ascii \"import gdb\\n\"\n"
